etd_app/templates/etd_app/candidate.html

```diff
@@ -6,29 +6,28 @@
 
 {% block content_main %}
     <h4>Dissertation Information</h4>
-{% if candidate.thesis %}
     <dl class="dl-horizontal">
-      <dt>Title: </dt><dd>{{ candidate.thesis.title }}</dd>
-    <dt>Abstract: </dt><dd>{{ candidate.thesis.abstract }}</dd>
-    
-    <dt>Keywords: </dt>{% for keyword in candidate.thesis.keywords.all %}
-    <dd>{{ keyword }} </dd>
-    {% endfor %}
-    {% if candidate.thesis.language %}
+      <dt>Title: </dt><dd>{{ candidate.thesis.title }}&nbsp;</dd>
+    <dt>Abstract: </dt><dd>{{ candidate.thesis.abstract }}&nbsp;</dd>
+    <dt>Keywords: </dt>
+      {% if candidate.thesis.keywords.all %}
+        {% for keyword in candidate.thesis.keywords.all %}
+          <dd>{{ keyword }}</dd>
+        {% endfor %}
+      {% else %}
+        <dd>&nbsp;</dd>
+      {% endif %}
     <dt>Language:</dt> <dd>{{ candidate.thesis.language.name }}</dd>
-    {% endif %}
-    <dt>Dissertation filename:</dt> <dd>{{ candidate.thesis.file_name }}</dd>
-    <dt>Dissertation status:</dt> <dd>{{candidate.thesis.get_status_display}}</dd>
+    <dt>Dissertation filename:</dt> <dd>{{ candidate.thesis.file_name }}&nbsp;</dd>
```

Comment:
Do you want to always put in the &nbsp;  ?
I sometimes use the default filter
It would look like ```{{ candidate.thesis.file|default:"&nbsp;"}}```